cinema/serializers.py

```diff
@@ -0,0 +1,18 @@
+from rest_framework import serializers
+
+from cinema.models import Movie
+
+
+class MovieSerializer(serializers.Serializer):
+    id = serializers.IntegerField(read_only=True)
+    title = serializers.CharField()
+    description = serializers.CharField()
```

Comment:
You've specified `CharField` for the `description` field in your serializer, but it's more reasonable to use `TextField` if you expect this field to contain a large amount of text. If you want to limit the number of characters, you should use `CharField` with the `max_length` option. Please, adjust your model accordingly.